site/specification/ORCv1.md

```diff
@@ -1155,6 +1155,9 @@ records non-null values, a DATA stream that records the number of
 seconds after 1 January 2015, and a SECONDARY stream that records the
 number of nanoseconds.
 
+* Note that if writer timezone is set, 1 January 2015 is according to
+this timezone and not according to UTC
```

Comment:
According to implementation:

https://github.com/apache/orc/blob/3b5b2a6286df48a4ab471aece74bc7b7947042ad/c%2B%2B/src/Timezone.hh#L66-L72

https://github.com/apache/orc/blob/9b79de995430b240cb68f225c22bef69ebbd8b07/c%2B%2B/src/ColumnReader.cc#L335

Also, I'm not certain if the writer timezone is mandatory in the stripe if a TIMESTAMP column is present in the file. Might need some clarification on this?

Reply:
IIUC, writer timezone field is mandatory if there is any timestamp type: https://github.com/apache/orc/blob/513922af6c1d1a07e632a03231d5b546eeafb26e/java/core/src/java/org/apache/orc/impl/WriterImpl.java#L559-L565

Reply:
Thanks, I think I'll add this to the spec as well (maybe proto too, in the other repo?)